CSV/49-2/VolcanicAshConcentrations/VolcanicAshConcentrations_entity.csv

```diff
@@ -0,0 +1,5 @@
+id,notation,status,description,label,altLabel,source,seeAlso,related,note,iwxxmVersionInfo
+http://codes.wmo.int/49-2/VolcanicAshConcentrations/LOW_AND_POSSIBLY_HIGHER,LOW_AND_POSSIBLY_HIGHER,experimental,Greater than or equal to 0.2 mg/m3 - Low and possibly higher,Greater than or equal to 0.2 mg/m3 - Low and possibly higher,,,,,,2025-2
```

Comment:
```suggestion
http://codes.wmo.int/49-2/VolcanicAshConcentrations/LOW_AND_POSSIBLY_HIGHER,LOW_AND_POSSIBLY_HIGHER,experimental,Greater than or equal to 0.2 mg/m3,Low and possibly higher,,,,,,2025-2
```

Reply:
> Agree that descriptions should not be just a copy of label which are not able to give additional information.

Yes, where possible for new codes, let's try to NOT repeat the label in the description.

Reply:
Two things:  First of all this code list should not exist under 49-2; it has been moved to iwxxm and it is my mistake for not being removed.  Secondly, it has been decided not to categorize concentration levels at all so this code list should be removed no matter under 49-2 or iwxxm.  See https://github.com/wmo-im/iwxxm/issues/343#issuecomment-2980074169 for details.